tools/RunUnity/Program.cs

```diff
@@ -34,6 +34,7 @@ static void Main(string[] args)
                 }
 
                 var path = Path.Combine(Paths.TryGetUnityPath(), location);
+                Console.WriteLine($"Found Unity in {path}");
```

Comment:
added this as I got trolled for roughly an hour by it picking the wrong Unity version...